### PR TITLE
This commit completes Task 9 - DELETE /api/comments/:comment_id

### PR DESCRIPTION
**Pull request by yaiqbal**

### CODE REVIEW

File: __tests__/index.test.js [hunk 1]
```diff
@@ -278,4 +278,29 @@ describe('/api/articles/:article_id', () => {
...
   })
+})
+
+
+describe('/api/comments/:comment_id', () => {
+  test('DELETE:204 deletes the specified comment and sends no content back', () => {
+    return request(app)
+      .delete('/api/comments/1')
+      .expect(204);
+  });
```
Comment (MKD01) [inline __tests__/index.test.js:289]: Nice first test

File: __tests__/index.test.js [hunk 2]
```diff
@@ -278,4 +278,29 @@ describe('/api/articles/:article_id', () => {
...
+      });
+  });
+  test('DELETE:400 responds with an appropriate status and error message when given an invalid id', () => {
+    return request(app)
+      .delete('/api/comments/not-a-team')
+      .expect(400)
+      .then((response) => {
+        expect(response.body.msg).toBe('Bad request');
+      });
+  });
```
Comment (MKD01) [inline __tests__/index.test.js:305]: Excellent error handling

File: controllers/news.controller.js [hunk 3]
```diff
@@ -85,4 +86,19 @@ exports.patchArticleController = (req, res, next) => {
...
+}
+
+exports.deleteCommentByIdController = (req, res, next) => {
+  const {comment_id} = req.params
+
+  deleteCommentByIdModel(comment_id)
+    .then(rowCount => {
+      if(!rowCount) {
+        return res.status(404).send({ msg: 'Comment does not exist' });
+      }
```
Comment (MKD01) [inline controllers/news.controller.js:98]: Error handling should be done in the model not the controller

File: controllers/news.controller.js [hunk 4]
```diff
@@ -85,4 +86,19 @@ exports.patchArticleController = (req, res, next) => {
...
+      res.status(204).send();
```
Comment (MKD01) [inline controllers/news.controller.js:99]: A nice alternative to this is `res.sendStatus(204)` which then doesnt need the empty .send at the end

**Review by MKD01 — COMMENTED** [review]
Nice work

**Comment by yaiqbal** [comment]
Thank you MKD for the review comments.

**Pull request merged**